cibuildwheel/linux.py

```diff
@@ -59,7 +62,11 @@ def build(project_dir, output_dir, test_command, test_requires, test_extras, bef
 
             {environment_exports}
 
+            $(cat $(which auditwheel) | head -1 | tail -c +3 | head -c -1) -m pip install --upgrade auditwheel>=3.1.0
```

Comment:
I still think that it is better to make own docker repository with upgraded version of audithwheel and wheel and use it in 
```
default_manylinux_images_pypy_x86_64 = {'manylinux2010': 'pypywheels/manylinux2010-pypy_x86_64'} 
```
tomorow may be next wheel release which will break API and auditwheel stops working

Reply:
Yeah, I was thinking I'll make a PR to pypy/manylinux, probably. I agree we shouldn't merge/release this PR before fixing that.

Reply:
`wheel==0.34.0` introduce new (proper) tag for pypy wheels. `auditwheel` use `wheel` api which is unstable so `auditwheel` is pinned to given  `wheel` release. Current release of pyp manylinux docker image contains old version of `auditwheel`. This line is temporary workaround to upgrade `auditwheel` package (If I properly understand previous @YannickJadoul comments here and in main discussion).

Reply:
The annoying thing is that the `python` that was used to install the `auditwheel` script is not on  `PATH` (`auditwheel` is only installed once, not for every `python` version in the image); so in order to update it, I look at the shebang of the `auditwheel` script to update it (as temporary hack). Yes, if `manylinux` updates, this won't do anything, I think (actually: no, it will still do anything because of the '--upgrade` flag), but I was thinking that I should make a PR to the pypy/manylinux repo, to get this fixed upstream.

Reply:
Do not forget about this.

Reply:
I am not forgetting. I am actually looking at this as we speak.
But first the build and simple changes need to pass.